shoop/themes/classic_gray/templates/classic_gray/shoop/front/product/detail.jinja

```diff
@@ -106,6 +109,23 @@
                     {% include "shoop/front/product/_detail_attributes_section.jinja" %}
                 </div>
             {% endif %}
+            {% if media_files %}
```

Comment:
This segment (112..128) looks suspiciously similar to the body of the `render_ordered_files` macro. Could they maybe be unified?